### DISCUSSION TITLE
Mounting options/3D models for heat meters

### DISCUSSION THREAD

**Discussion by DanielWe2** [jomjol/AI-on-the-edge-device#1115] [General]
Hello,

I want to use this project to monitor my heat meter (Wärmemengenzähler) which is just loosely laying on the floor. 

What is the optimal distance between the meter and the cam?  And what are your ideas on securely mounting the ESP to the heat meter?  

A picture of my unit:

![grafik](https://user-images.githubusercontent.com/19294888/193116450-26390180-f61d-45b6-9600-b3d073dc803b.png)

**Comment by purehunter1** [comment]
If the cameras and lenses on the ESP32-CAM are all the same, I can say as a result of my measurements, the distance shall be about (width of the digits)*1.5 as an absolute minimum.
More distant is better: less image distortion, easier camera positioning.
Best mounting option is a 3D print. The housing must be opaque / completely dark inside.

**Reply by caco3** [reply]
There are also wide angle cameras (120°) which can be mounted closer (and are supported). How ever I saw that it generates some fish eye distortion, therefore I did not test if it actually would work!

**Comment by DanielWe2** [comment]
Thanks everyone. I designed some simple to mount the esp to the meter.  I am using the model from @nliaudat  from #1025 for the esp itself and my model to connect it to the actual meter.

I couldn't get a clear image with the integrated LED and added two sk6812 LEDs to get rid of the reflections. (@jomjol You can remove the warning about SK6812 not been tested, they work just fine).

The result looks like this:

![20221004_005011](https://user-images.githubusercontent.com/19294888/194408055-12ec6cdf-f16b-4a08-bf1b-ffd9cb062454.jpg)

The quality of the image is not that great but it seems to recognize the correct numbers:

![Screenshot_20221004_005358](https://user-images.githubusercontent.com/19294888/194409019-f3d76f99-8181-45d2-9e35-bdf15d775559.jpg)

**Reply by caco3** [reply]
> You can remove the warning about SK6812 not been tested, they work just fine).

Done
